python/data_process.py

```diff
@@ -0,0 +1,71 @@
+import re
+from os import chdir, getcwd
+wd=getcwd()
+chdir(wd)
```

Comment:
It's a better practice to use os.path.join(directory, filename) in the code, instead of changing the wd in the script directly which might be more prone to errors.

Reply:
Thank you, I will change that!